api/payment.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"fmt"
+	"isar-golang-training-beginner/db"
+	"isar-golang-training-beginner/model"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/labstack/echo"
+)
+
+const (
+	ActiveStatus   = "ACTIVE"
+	IncativeStatus = "INACTIVE"
+	ExpiredStatus  = "EXPIRED"
```

Comment:
Do you know the different between `ActiveStatus` and `activeStatus`?

If not, try to change it and call from different package (e.g. from package config)

Reply:
noted, will update this to include in task 3